src/ucp/wireup/wireup_ep.c

```diff
@@ -389,8 +377,12 @@ static UCS_CLASS_CLEANUP_FUNC(ucp_wireup_ep_t)
     if (self->aux_ep != NULL) {
         ucp_worker_iface_unprogress_ep(ucp_worker_iface(worker,
                                                         self->aux_rsc_index));
+        uct_ep_pending_purge(self->aux_ep, ucp_wireup_replay_pending_request,
```

Comment:
i think it's safer to gather all pending reqs on a list and then replay them: what if some request tries to access aux_ep while it's being destroyed?